framework/src/modules/interoperability/base_interoperability_method.ts

```diff
@@ -143,8 +146,22 @@ export abstract class BaseInteroperabilityMethod<
 		);
 	}
 
-	// eslint-disable-next-line @typescript-eslint/require-await
-	public async terminateChain(_methodContext: MethodContext, _chainID: Buffer): Promise<void> {
-		throw new Error('Need to be implemented');
+	public async terminateChain(context: TerminateChainContext, chainID: Buffer): Promise<void> {
+		if (await this.getTerminatedStateAccount(context, chainID)) {
+			return;
+		}
+
+		await this.internalMethod.sendInternal(
+			context,
+			EMPTY_FEE_ADDRESS,
+			MODULE_NAME_INTEROPERABILITY,
+			CROSS_CHAIN_COMMAND_NAME_CHANNEL_TERMINATED,
```

Comment:
maybe this can be renamed to `CROSS_CHAIN_COMMAND_NAME_CHANNEL_TERMINATED`

Reply:
It's already called `CROSS_CHAIN_COMMAND_NAME_CHANNEL_TERMINATED`

Reply:
ah sorry I meant `CROSS_CHAIN_COMMAND_CHANNEL_TERMINATED` as @sitetester mentioned